.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/ruby
 
 # Generated js bundles
 /app/assets/javascripts/generated/*
+
+# Rubymine/IntelliJ
+.idea
```

Comment:
> 1 - If someone comes along to contribute and they don't have a personal gitignore, they might accidentally commit files in .idea to this project.
That's true, even we have some of it, they might use some different environment on their local, so we better to leave that. They should better create a clean pull-request as well, so it's not good to create pull request and let maintainer think and clean up for them.

> 2 - I notice you have .ruby-version in your global gitignore above. What if you're working a project that wants that committed?
Because it's not always, but when the product need it to be included, then we could use `git add -f` for the first time, then git will tracking this file as like other files.